libraries/opensk/src/api/crypto/hmac256.rs

```diff
@@ -20,9 +20,15 @@ pub trait Hmac256 {
     fn mac(key: &[u8; HMAC_KEY_SIZE], data: &[u8]) -> [u8; HASH_SIZE];
 
     /// Verifies the HMAC.
+    ///
+    /// When implementing, make sure that you don't leak side channel information about the key,
+    /// i.e. by using constant time comparisons of the computed and given MAC.
```

Comment:
nit: descriptive documentation

```rust
/// This function does best effort to not leak information about the key through side-channels
/// (e.g. usage of constant time comparison).
```

It is implicit that what is guaranteed by the function is what implementations must satisfy.

But no need to change.